mentors/java/d.json

```
@@ -22,5 +22,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I am developer mostly practicing Java, Scala, clean code and TDD"
-  }  
+  },
+  {
+    "github_username": "danruff5",
+    "name": "Daniel Krauskopf",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I graduadeted from Fanshawe college with two programming degrees. I am currentally a full-time employee for a large company using Java and many other technologies."
```

Just a few misspellings that need fixing:
```suggestion
    "bio": "I graduated from Fanshawe College with two programming degrees. I am currently a full-time employee for a large company using Java and many other technologies."
```